twisted/test/test_defer.py

```diff
@@ -2446,3 +2446,20 @@ def test_cancelDeferUntilLockedWithTimeout(self):
         self.assertFalse(timeoutCall.active())
         self.assertIsNone(self.lock._timeoutCall)
         self.failureResultOf(deferred, defer.CancelledError)
+
+
+
+class TimeoutErrorTests(unittest.TestCase):
+    def test_deprecatedTimeoutError(self):
+        """
+        Test that L{twisted.internet.defer.TimeoutError} is deprecated.
```

Comment:
The docstring can be 

```
L{twisted.internet.defer.TimeoutError} is deprecated.
```

as `Test that` is assumed to be present in the docstring of each test